File: array/contains-duplicate.js
/**
 * https://leetcode.com/problems/contains-duplicate
 * 
 * @param {number[]} nums
 * @return {boolean}
 */
const containsDuplicate = (nums) => { 
    if ([...new Set(nums)].length < nums.length) {
        return true;
    }
    return false;
};

File: cplusplus/array/contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        set<int> seenNums;

        for (int i : nums) {
            if (seenNums.contains(i)) {
                return true;
            } else {
                seenNums.insert(i);
            }
        };

        return false;
    }
};